app/uk/gov/hmrc/gform/sharedmodel/formtemplate/destinations/Destination.scala

```diff
@@ -81,10 +79,12 @@ object Destination {
     uri: String,
     method: HttpMethod,
     payload: Option[String],
+    payloadName: Option[String],
```

Comment:
`payloadName` should be derived. It should equal to `formTemplateId-destinationId`